dist/index.js

```diff
@@ -7300,19 +7300,13 @@ async function maybeInstallRebar3(rebar3Spec) {
 
 async function getOTPVersion(otpSpec0, osVersion) {
   const otpVersions = await getOTPVersions(osVersion)
-  const otpSpec = otpSpec0.match(/^(OTP-|maint-)?([^ ]+)/)
-  let otpVersion
-  if (otpSpec[1] && !isStrictVersion()) {
-    throw new Error(
-      `Requested Erlang/OTP version (${otpSpec0}) ` +
-        "should not contain 'OTP-, or maint-'",
-    )
-  }
-  if (otpSpec) {
-    otpVersion = getVersionFromSpec(
-      otpSpec[2],
-      Array.from(otpVersions.keys()).sort(),
-    )
+  let otpSpec = otpSpec0 // might be a branch (?)
```

Comment:
Greatly simplified. We drop OTP-, maint- and whatnot to favour "branch over version," as suggested by @ericmj.